quiz-microsevice/business-logic/services/quiz-service.js

```diff
@@ -196,4 +196,37 @@ const createQuiz = async (
   return { ...quiz, questions: questionsWithoutQuizId };
 };
 
-module.exports = { createQuiz };
+const retrieveQuizzes = async (clientId) => {  
```

Comment:
Take the client object it self as parameter like createQuiz then send just the id to repository.